tools/offsets-tool-py/offsets-tool.py

```diff
@@ -183,7 +225,7 @@ def run_clang(self):
 							continue
 						if child.is_bitfield ():
 							continue
-						rtype.fields.append (FieldInfo (child.spelling, child.get_field_offsetof () / 8))
+						rtype.fields.append (FieldInfo (child.spelling, int (child.get_field_offsetof () / 8)))
```

Comment:
on Python3 `child.get_field_offsetof () / 8` yields a float, so you would have entries like this:
```c
DECL_OFFSET2(MonoClass,interface_offsets_count,100.0)
```

Reply:
Could use // for division.